ext/openssl/openssl.stub.php

```diff
@@ -2,61 +2,93 @@
 
 /** @generate-function-entries */
 
+final class X509Certificate
+{
+}
+
+final class X509CertificateSigningRequest
```

Comment:
Probably it's a bit loo long a class name... I don't like X509CSR either, because it's crazy cryptic.

Reply:
I think better long and descriptive than short and cryptic. IDEs do have code completion.